### PR TITLE
[stable/neo4j] minor improvements to charts

### PR DESCRIPTION
**Pull request by rjaxin**
-Changed the empty values in values.yaml to arrays instead of dicts.  This prevents following error when overriding them from a values parent chart:
```
warning: cannot overwrite table with non table for sidecarContainers (map[])
warning: cannot overwrite table with non table for initContainers (map[])
```

-Added variable for sidecarContainers for replicas as well

### CODE REVIEW

**Comment by mneedham** [comment]
LGTM

File: stable/neo4j/templates/readreplicas-deployment.yaml [hunk 1]
```diff
@@ -66,6 +66,9 @@ spec:
           mountPath: /plugins
         securityContext:
           privileged: true
+{{- if .Values.core.sidecarContainers }}
+{{ toYaml .Values.core.sidecarContainers | indent 6}}
```
Comment (unguiculus) [inline stable/neo4j/templates/readreplicas-deployment.yaml]: Please add a space before `}}`.

Comment (rjaxin) [reply stable/neo4j/templates/readreplicas-deployment.yaml]: @unguiculus - Thanks, I have addressed this suggestion.

**Pull request merged**